target_benchmark/tasks/AbsTask.py

```diff
@@ -449,6 +532,17 @@ def _calculate_table_retrieval_performance(
         Returns:
             a retrieval performance data model that contains the accuracy of the retrieval for a dataset on this task.
         """
+        retrieval_duration_process = round(total_retrieval_duration_process, 5)
+        retrieval_duration_wall_clock = round(total_retrieval_duration_wall_clock, 5)
+        avg_dur_process = 0
+        avg_dur_wall_clock = 0
+        if not total_num_queries:
```

Comment:
This could potentially lead to some sneaky bugs down the line - is `total_num_queries` an optional argument that could be `None`? Or are you checking if the value of the variable is equal to `0`? 

If the latter, which from the usage in the codebase it seems to be, I'd highly encourage a rewrite to:

```suggestion
        if total_num_queries == 0:
```

Also - why is this annotated as a `float`? My interpretation would be as an `int`?

Reply:
you are right! it might have been a mixup with another variable of the same name in the `task_run` function. I changed the name to improve clarity.